pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## Summary
+
+<!-- Required section. Describe what this PR will do if it is merged. -->
+
+## Details
```

Comment:
"Details" seems maybe redundant to "Summary" and the PR title? 

what do you think about the sample here? https://eng-handbook.hashicorp.services/git-and-github/#templates

Reply:
I don't agree that they are the same thing. Often a summary should suffice; however there are times when a PR requires more detailed information be added for context. That's why this section is optional and can be removed.

Reply:
sure sounds good! i typically push that summary/detail hierarchy to the commit message. i often just copy my commit messages to the PR summary

here's an example of what i mean from https://eng-handbook.hashicorp.services/git-and-github/#commit-messages 

```
Example commit message:

feature: Allow users to navigate to X from the sidebar

This feature was necessary since a customer wanted to be able to quickly access X while in the UI. We will probably want to break out this feature into the component library so other products can use it as well.
```